website/static/js/newAndNoteworthyPlugin.js

```diff
@@ -22,6 +22,12 @@ var NewAndNoteworthy = {
         self.popularNodes = m.prop([]);
         self.contributorsMapping = {};
         self.SHOW_TOTAL = 5;
+        self.errorLoading = false;
+
+        // Switches errorLoading to true
+        self.requestError = function(){
+            self.errorLoading = true;
```

Comment:
This works for the user but maybe we should send an error to sentry as well with something like this 
```
self.requestError = function(results){
    var message = 'Error loading project category names.';
    Raven.captureMessage(message, {requestReturn: results});
};
```